boards/arduino/opta/arduino_opta-common.dtsi

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2023 Felipe Neves
+ * Copyright (c) 2024 DNDG srl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	leds {
+		compatible = "gpio-leds";
+
+		status_led_1: led_1 {
+			gpios = <&gpioi 0 GPIO_ACTIVE_HIGH>;
+		};
+		status_led_2: led_2 {
+			gpios = <&gpioi 1 GPIO_ACTIVE_HIGH>;
+		};
+		status_led_3: led_3 {
+			gpios = <&gpioi 3 GPIO_ACTIVE_HIGH>;
+		};
+		status_led_4: led_4 {
+			gpios = <&gpioh 15 GPIO_ACTIVE_HIGH>;
+		};
+		led_reset: led_g {
+			gpios = <&gpioh 12 GPIO_ACTIVE_HIGH>;
+		};
+		led_reset_red: led_r {
+			gpios = <&gpioh 11 GPIO_ACTIVE_HIGH>;
+		};
+		led_user: led_b {
+			gpios = <&gpioe 5 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	relays {
+		compatible = "gpio-power-switches";
+
+        output_d0: d0 {
```

Comment:
this file's formatting needs sorting

Reply:
What do you mean by "sorting"? I see that I used spaces in a file that originally used tabs: fixed that but I don't if it was what you wanted.

Reply:
That was it